src/api-reference/budget/budget-header.md

```diff
@@ -227,7 +227,7 @@ Name | Description
 
 Name | Type | Format | Description
 -----|------|--------|------------			
-id	|	`string`	|	`path`	|	The budget item header's key field (sync guid).
+id	|	`string`	|	`path`	|	The budget item header's key field.
```

Comment:
Let's go ahead and cleanup these table whitespace issues.

Reply:
👍